meeshkan/serve/mock/storage.py

```diff
@@ -1,28 +1,106 @@
 import logging
+import uuid
+
+from http_types import Request
+from openapi_typed import OpenAPIObject
 
 logger = logging.getLogger(__name__)
 
 
-class StorageManager:
+class Storage:
     def __init__(self):
         self._default = dict()
-        self._storages = dict()
+        self._entities = dict()
```

Comment:
Please add some typing or docstring here to explain what are the expected keys and values?

Reply:
Also the class could use a docstring explaining what it's meant for.